modules/accredited_representative_portal/app/sidekiq/accredited_representative_portal/replicator.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+module AccreditedRepresentativePortal
+  class Replicator
+    class ValidationError < StandardError; end
+
+    Result = Struct.new(:deleted_records, :inserted_records, keyword_init: true)
```

Comment:
Maybe let's not bother with results unless it's easy to return something with clear fidelity from AR.